firefox-ios/nimbus-features/tabTrayUIExperiments.yaml

```
@@ -9,11 +9,18 @@ features:
           If true, enables the tab tray UI experiments.
         type: Boolean
         default: false
+      animation-feature:
+        description: >
+          If true, enabes the tab tray animation feature.
```

Spelling, and might be helpful to be more explicit on the animation.

```suggestion
          If true, enables the tab tray animation feature, that occurs when the tab tray opens and closes.